SteamKit2/SteamKit2/SteamKit2.csproj

```diff
@@ -16,6 +16,11 @@
   <PropertyGroup>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\SteamKit2.xml</DocumentationFile>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Perhaps this should just be `Condition="'$(Configuration)'=='Debug'"`. I know Visual Studio generates it in this form, but platform is unneccesary here.